config/application.yml.default

```diff
@@ -305,13 +282,13 @@ production:
   no_sp_campaigns_whitelist: '[]'
   nonessential_email_banlist: '[]'
   otp_delivery_blocklist_findtime: '5'
+  otp_use_alphanumeric: 'false'
```

Comment:
here we is override to `false` in prod. once this PR is promoted, I'd like to immediately flip this flag to `true` and recycle again (so that we ensure all boxes can accept alphanumeric OTPs before we start sending alphanumeric OTPs)